Bluejay/Bluejay/Bluejay.swift

```diff
@@ -19,7 +19,7 @@ public class Bluejay: NSObject {
     // MARK: - Private Properties
     
     /// Internal reference to CoreBluetooth's CBCentralManager.
-    private var cbCentralManager: CBCentralManager!
+    public var cbCentralManager: CBCentralManager!
```

Comment:
I know this has already been approved... however I would inhibit setting this property from outside, at least unless that's the intended goal.
```suggestion
    private(set) public var cbCentralManager: CBCentralManager!
```